msteams-platform/resources/bot-v3/bot-conversations/bots-conversations.md

```diff
@@ -152,7 +152,7 @@ When interacting in a channel, your bot should be smart about taking certain con
 
 The `channelData` object contains Teams-specific information and is the definitive source for team and channel IDs. You should cache and use these ids as keys for local storage.
 
-The `channelData` object is not included in messages in personal conversations since these take place outside of any channel.
```

Comment:
@v-npaladugu - Could you please remove the below line from documentation:
Doc section for ref : https://docs.microsoft.com/en-us/microsoftteams/platform/resources/bot-v3/bot-conversations/bots-conversations#teams-channel-data

UI
![image](https://user-images.githubusercontent.com/59052547/146310375-34980b4e-7523-4e34-9d4a-ebde34ce2d82.png)